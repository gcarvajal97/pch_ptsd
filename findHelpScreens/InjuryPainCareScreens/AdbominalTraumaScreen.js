import React, { Component } from 'react';
import { View, Text, StyleSheet } from 'react-native';
import translate from '../../components/translateService';
import colors from '../../colors';

export default class AdbominalTraumaScreen extends Component {
  render() {
    return (
      <View>
        <Text style={styles.paragraph}>{translate('injuryPainCare.Ch2_p1')}</Text>
        <Text style={styles.paragraph}>{translate('injuryPainCare.Ch2_p2')}</Text>

        <Text style={styles.paragraphTitle}>{translate('injuryPainCare.CH2_TITLE_1')}</Text>
        <Text style={styles.paragraph}>{translate('injuryPainCare.Ch2_t1_b1')}</Text>      
        <Text style={styles.paragraph}>{translate('injuryPainCare.Ch2_t1_b2')}</Text>
        <Text style={styles.paragraph}>{translate('injuryPainCare.Ch2_t1_b3')}</Text>
        <Text style={styles.paragraph}>{translate('injuryPainCare.Ch2_t1_b4')}</Text>
        <Text style={styles.paragraph}>{translate('injuryPainCare.Ch2_t1_b5')}</Text>
                
        <Text style={styles.paragraphTitle}>{translate('injuryPainCare.CH2_TITLE_2')}</Text>
        <Text style={styles.paragraph}>{translate('injuryPainCare.Ch2_t2_b1')}</Text>
        <Text style={styles.paragraph}>{translate('injuryPainCare.Ch2_t2_b2')}</Text>
        <Text style={styles.paragraph}>{translate('injuryPainCare.Ch2_t2_b3')}</Text>
        <Text style={styles.paragraph}>{translate('injuryPainCare.Ch2_t2_b4')}</Text>
        <Text style={styles.paragraph}>{translate('injuryPainCare.Ch2_t2_b5')}</Text>
        <Text style={styles.paragraph}>{translate('injuryPainCare.Ch2_t2_b6')}</Text>
        <Text style={styles.paragraph}>{translate('injuryPainCare.Ch2_t2_b7')}</Text>
        <Text style={styles.paragraph}>{translate('injuryPainCare.Ch2_t2_b8')}</Text>
      </View>
    );
  }
}

const styles = StyleSheet.create({
  paragraphTitle: {
    fontFamily: "avenir-heavy",
    fontSize: 20,
    alignSelf: 'stretch',
    textAlign: 'left',
    marginTop: 6,
    marginBottom: 3,
    color: colors.accent
  },
  paragraph: {
    textAlign: 'left',
    fontSize: 14,
    fontFamily: "avenir-medium",
    marginVertical: 2,
  },
  paragraphBold: {
    textAlign: 'left',
    fontSize: 14,
    fontFamily: "avenir-heavy",
    marginVertical: 2,
    marginLeft: 10
  }
});