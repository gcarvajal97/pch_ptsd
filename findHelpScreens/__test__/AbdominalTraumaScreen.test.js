import React from 'react';
import AdbominalTraumaScreen from '../InjuryPainCareScreens/AdbominalTraumaScreen';

import renderer from 'react-test-renderer';

test('renders correctly', () => {
    const tree = renderer.create(<AdbominalTraumaScreen/>).toJSON();
    expect(tree).toMatchSnapshot();
}) 