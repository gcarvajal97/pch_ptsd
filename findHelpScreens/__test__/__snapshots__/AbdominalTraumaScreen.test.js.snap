// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View>
  <Text
    style={
      Object {
        "fontFamily": "avenir-medium",
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    Abdominal trauma involves injury to the belly between the chest and the hips and the internal organs there â€“ the most common injuries to organs in the belly include the small intestine, liver, spleen, pancreas, kidneys, and bladder.
  </Text>
  <Text
    style={
      Object {
        "fontFamily": "avenir-medium",
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    Follow instructions provided by your healthcare provider (doctor or nurse practitioner). It is important that your child follow all of the special instructions given by your healthcare provider to decrease any risk of bleeding inside.
  </Text>
  <Text
    style={
      Object {
        "alignSelf": "stretch",
        "color": "#0077C8",
        "fontFamily": "avenir-heavy",
        "fontSize": 20,
        "marginBottom": 3,
        "marginTop": 6,
        "textAlign": "left",
      }
    }
  >
    Make sure to ask your healthcare provider:
  </Text>
  <Text
    style={
      Object {
        "fontFamily": "avenir-medium",
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - How long should my child only have minimal activity or 'house arrest'? This will generally depend on the grade of injury to your child's organ or how severe it was. Your doctor will tell you what minimal activity means. It could include activities such as quiet play, reading, coloring, and watching TV. It probably means that your child will have to avoid sports for a period of time.
  </Text>
  <Text
    style={
      Object {
        "fontFamily": "avenir-medium",
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - When can my child return to school?
  </Text>
  <Text
    style={
      Object {
        "fontFamily": "avenir-medium",
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - When can my child start gym? contact sports? biking? or rough play?
  </Text>
  <Text
    style={
      Object {
        "fontFamily": "avenir-medium",
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - This will also depend on the grade of injury to your child's organ or how severe it was.
  </Text>
  <Text
    style={
      Object {
        "fontFamily": "avenir-medium",
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - What is the best number to call for emergencies during the week? On weekends?
  </Text>
  <Text
    style={
      Object {
        "alignSelf": "stretch",
        "color": "#0077C8",
        "fontFamily": "avenir-heavy",
        "fontSize": 20,
        "marginBottom": 3,
        "marginTop": 6,
        "textAlign": "left",
      }
    }
  >
    When to call your healthcare provider:
  </Text>
  <Text
    style={
      Object {
        "fontFamily": "avenir-medium",
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - Ask your healthcare provider for specific signs and symptoms that should alert you to call them right away. They may include the following: Signs of intra-abdominal bleeding (bleeding in the belly) such as:
  </Text>
  <Text
    style={
      Object {
        "fontFamily": "avenir-medium",
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - Looking pale
  </Text>
  <Text
    style={
      Object {
        "fontFamily": "avenir-medium",
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - Lethargy (hard to keep awake)
  </Text>
  <Text
    style={
      Object {
        "fontFamily": "avenir-medium",
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - Blood in urine, stools (bowel movement) or vomitus (throw up)
  </Text>
  <Text
    style={
      Object {
        "fontFamily": "avenir-medium",
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - Fever (temperature greater than 101 degrees Fahrenheit or 38.5 degrees Celsius)
  </Text>
  <Text
    style={
      Object {
        "fontFamily": "avenir-medium",
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - Increased pain
  </Text>
  <Text
    style={
      Object {
        "fontFamily": "avenir-medium",
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - Vomiting (or throwing up)
  </Text>
  <Text
    style={
      Object {
        "fontFamily": "avenir-medium",
        "fontSize": 14,
        "marginVertical": 2,
        "textAlign": "left",
      }
    }
  >
    - Any other concerns that you may have.
  </Text>
</View>
`;
